expected/line.out

```diff
@@ -134,4 +134,35 @@ SELECT sline ( spoint '(0,   0d)', spoint '(0.000001d, 0d)' )   #
  f
 (1 row)
 
-       
+-- checking the distance between a line and a point
+SELECT round( CAST(sline '( 90d, 0d, 0d, XYZ ), 40d ' <-> spoint '( 0d, 90d )' as numeric), 14);
+      round       
+------------------
+ 0.87266462599717
+(1 row)
+
+SELECT round( CAST(sline '( 90d, 0d, 0d, XYZ ), 40d ' <-> spoint '( 0d, 90d )' as numeric), 14)   =
```

Comment:
I think round(CAST... is not necessary.
I would propose to add SET extra_float_digits TO -3; in the beginning of file. It reduces the number of output digits.

I've attached the modified line.sql 
[line.sql.zip](https://github.com/postgrespro/pgsphere/files/12266486/line.sql.zip)